Cargo.toml

```diff
@@ -33,3 +34,7 @@ init = []          # Enables automatically choosing a camera
 [[example]]
 name = "3d_scene"
 path = "examples/3d_scene.rs"
+
+[[example]]
```

Comment:
I wondering why the original author included the `3d_scene` example in Cargo.toml. They are discovered automatically anyway.